packages/main/src/components/protected/topbar/components/Profile.jsx

```diff
@@ -42,7 +42,7 @@ const Profile = () => {
   useEffect(() => {
     if (currentWorkspace) {
       authAxios
-        .get(`/organizations/${currentWorkspace}`)
+        .get(`v`)
```

Comment:
Where is this `v` coming from?

Reply:
I seriously didn't touch any Axios call. I don't know how it happens to be my changes, probably when I was merging. I was only working on the data that should be in currentWorkspace and when to use the long URL for axios. But the way it is, I don't think it is needed, so I will remove it.

Reply:
I just reviewed the code very well, I think probably my keyboard or something, but it wasn't intentional. So, I have reverted back to this